deploy/vulnerability/cloudbeat-vuln-mgmt.yml

```diff
@@ -38,9 +38,12 @@ output.elasticsearch:
 
 # ================================= Processors =================================
 processors:
-  - add_host_metadata: ~
-  - add_cloud_metadata: ~
-  - add_docker_metadata: ~
```

Comment:
Basically the fields added by these processors are irrelevant for CNVM.
Cloudbeat populates the same fields with values of the scanned object rather than the host (the scanner instance).
It also doesn't run in a docker.

Reply:
will this affect the users? or does this is only for dev purposes and what really matter is the one in /integrations